_sass/components/_section.scss

```diff
@@ -39,8 +45,4 @@ header {
     margin-top: (_size(element-margin) * -0.3);
     line-height: 1.5em;
   }
-
-  &.image-header h1 {
```

Comment:
We'll want to keep this, it keeps the logo's header text hidden.  Rationale for having a hidden header is for SEO purposes

Reply:
I was not aware that I had edited this at all, was this changed between earlier commit and the revert I did?

Reply:
either way will revert to having this again